TransportationSplitter.py

```diff
@@ -467,12 +662,32 @@ def make_split_point(connector, segment_geometry):
     ])
 )
 
+resolved_destinations_schema = ArrayType(
+    StructType([
+        StructField('labels', ArrayType(
+            StructType([
+                StructField('value', StringType(), True), 
+                StructField('type', StringType(), True)
+                ]), True), 
+            True), 
+        StructField('symbols', ArrayType(StringType(), True), True), 
+        StructField('from_connector_id', StringType(), True), 
+        StructField('to_segment_id', StringType(), True), 
+        StructField('to_segment_start_lr', DoubleType(), True), 
+        StructField('to_segment_end_lr', DoubleType(), True), 
+        StructField('to_connector_id', StringType(), True), 
+        StructField('when', StructType([
+            StructField('heading', StringType(), True)
+            ]), True), 
+        StructField('final_heading', StringType(), True)
+        ]), False)
+
 def split_joined_segments(df: DataFrame, lr_columns_for_splitting: list[str]) -> DataFrame:
     broadcast_lr_columns_for_splitting = sc.broadcast(lr_columns_for_splitting)    
     input_fields_to_drop_in_splits = ["joined_connectors"]
     transportation_feature_schema = StructType([field for field in df.schema.fields if field.name not in input_fields_to_drop_in_splits])
     split_segment_fields = [field for field in transportation_feature_schema] + additional_fields_in_split_segments
-    if PROHIBITED_TRANSITIONS_COLUMN in lr_columns_for_splitting:
+    if PROHIBITED_TRANSITIONS_COLUMN in df.columns:
```

Comment:
this _if in  lr_columns_for_splitting_ was wrong, the prohibited_transitions references need to be resolved to have start/end lr no matter what, because the target segment ids do get split.

Reply:
this should be fixed now